tasks/send-slack-notification/README.md

```diff
@@ -0,0 +1,16 @@
+# slack-webhook-notification task
+
+This task sends a message to Slack using incoming webhook
+
+## Parameters
+| Name         | Description                                                                                                                     | Optional | Default Value                     |
+|--------------|---------------------------------------------------------------------------------------------------------------------------------|----------|-----------------------------------|
+| message      | Message to be sent                                                                                                              | false    |                                   |
+| secretName   | Name of secret which contains key/value pairs of `<team>/<webhook url>`. e.g. team1=https://hooks.slack.com/services/XXX/XXXXXX | true     | slack-webhook-notification-secret |
+| dataJsonPath | Path to data json file. It should contain a key called 'slack-webhook-notification-secret-keyname'                              | true     | $(workspaces.data.path)/data.json |
```

Comment:
this is not a good default value. The file in the pipelines will be in workspace/subdir/data.json. It barely makes sense for us to have `data.json` as defaults (but helps with our tests). workspace/data.json does not make sense though